docs/modules/getting-started/pages/blue-green.adoc

```diff
@@ -0,0 +1,255 @@
+    <clients>
+        <client>client-blue.xml</client>
+        <client>client-green.xml</client>
+    </clients>
+</hazelcast-client-failover>
+----
+====
++
+In this failover configuration file, you are directing the client to connect to the clusters in the given order from top to bottom;
+see xref:clients:java#ordering-of-clusters-when-clients-try-to-connect[Ordering of Clusters]. So, when you start the client
+(see Step 4 below), it will initially connect to the blue cluster. Here is what may happen:
+
+* When the blue cluster is gone, the client attempts to reconnect to it four times.
+* If not successful, it will try to connect to the green cluster, again, four times.
+* If these eight attempts are not successful, the client is shutdown.
+
+== Step 4. Connect the Client to Blue Cluster
+
+In this step, you'll start the client.
+
+[source,java]
+----
+HazelcastInstance client = HazelcastClient.newHazelcastFailoverClient();
+----
+
+Assuming that the blue cluster is alive, you should see a log similar to the following on the blue cluster’s terminal, showing that the client is connected.
+
+[source,shell]
+----
+2021-12-01 18:11:33,928 [ INFO] [hz.wizardly_taussig.priority-generic-operation.thread-0] [c.h.c.i.p.t.AuthenticationMessageTask]: [172.18.0.2]:5701 [blue] [5.0] Received auth from Connection[id=5, /172.18.0.2:5701->/172.18.0.1:61254, qualifier=null, endpoint=[172.18.0.1]:61254, alive=true, connectionType=JVM, planeIndex=-1], successfully authenticated, clientUuid: bf2ba9e2-d6f5-4a63-af43-e8d5ed8174b4, client name: hz.client_1, client version: 5.0
+----
+
+You can also verify the client is connected on the client side's terminal.
+
+[source,shell]
+----
+INFO: hz.client_1 [blue] [5.0] Trying to connect to [172.18.0.2]:5701
+Dec 01, 2021 8:11:33 PM com.hazelcast.core.LifecycleService
+INFO: hz.client_1 [blue] [5.0] HazelcastClient 5.0 (20210922 - dbaeffe) is CLIENT_CONNECTED
+----
+
+== Step 5. Kill the Blue Cluster
+
+Now, you'll kill the blue cluster and see the client is automatically connected to the green one.
+
+. Shutdown the `blue` cluster on its terminal simply by pressing **Ctrl+C**.
+. Verify that the client is connected to the green cluster on the cluster's and client's terminal.
++
+[source,shell]
+----
+2021-12-01 18:11:33,928 [ INFO] [hz.wizardly_taussig.priority-generic-operation.thread-0] [c.h.c.i.p.t.AuthenticationMessageTask]: [172.18.0.3]:5701 [green] [5.0] Received auth from Connection[id=5, /172.18.0.3:5701->/172.18.0.2:62432, qualifier=null, endpoint=[172.18.0.2]:62432, alive=true, connectionType=JVM, planeIndex=-1], successfully authenticated, clientUuid: bf2ba9e2-d6f5-4a63-af43-e8d5ed8174b4, client name: hz.client_1, client version: 5.0
+----
++
+[source,shell]
+----
+INFO: hz.client_1 [green] [5.0] Trying to connect to [172.18.0.3]:5701
+Dec 01, 2021 8:16:45 PM com.hazelcast.core.LifecycleService
+INFO: hz.client_1 [green] [5.0] HazelcastClient 5.0 (20210922 - dbaeffe) is CLIENT_CONNECTED
+----
+
```

Comment:
Needs' Next Steps'.

All of these tutorials would probably also benefit from a 'Clean Up' section. Since they all start new clusters, we should tell users how to remove the cluster if necessary so they can start afresh in the next tutorial.